secure_api/Crypto.AEAD.Main.fsti

```diff
@@ -300,5 +301,5 @@ val decrypt
   (ensures (fun h0 verified h1 ->
                invariant st h1 /\
                enc_dec_liveness st aad plain cipher_tag h1 /\
-               // modifies_fp FStar.Pointer.Base.(loc_union (footprint st) (loc_buffer (Plain.as_buffer plain))) h0 h1 /\
+               // modifies_fp FPLOC.(loc_union (footprint st) (loc_buffer (Plain.as_buffer plain))) h0 h1 /\
```

Comment:
where is `FPLOC` defined?

Reply:
I guess it's just an abstract reference for later on?

Reply:
yes, it could be replaced with FStar.Modifies.